webapp/python/pyproject.toml

```
@@ -0,0 +1,12 @@
+[project]
+name = "private-isu"
+version = "0.1.0"
+description = "private-isu webapp implementation for Python"
+readme = "README.md"
+requires-python = ">=3.13"
+dependencies = [
+    "flask>=3.1.0",
+    "gunicorn>=23.0.0",
+    "mysqlclient>=2.2.6",
```

今年になって 2.2.7 が出ているようでした。
動作確認は後ほど行います

```suggestion
    "mysqlclient>=2.2.7",
```